spec/files/test-variables.xql

```diff
@@ -0,0 +1,6 @@
+xquery version "3.0";
+
+declare option exist:serialize "method=json media-type=text/javascript";
+
+(: return value of $variable that should have been set in the query parameters:)
+<result>{$variable}</result>
```

Comment:
If this gets serialised to JSON anyway you could either just return $variable or wrap it in a `map { "variable": $variable }`.
What do you think?

Reply:
good point